src/main/java/com/google/devtools/build/lib/bazel/rules/android/AndroidNdkRepositoryFunction.java

```diff
@@ -398,6 +398,10 @@ public boolean verifyMarkerData(Rule rule, Map<String, String> markerData, Envir
             .replaceAll(
                 "%big_conditional_populating_variables%",
                 Joiner.on("\n" + "    ").join(bigConditional.build())));
+
+    writeFile(outputDirectory,
+        "platform_mappings",
```

Comment:
Does this actually work? Can the mappings processor read this?

Reply:
No it doesn't, I forgot to remove this after testing it out. It'd be nice to be able to read this, though.